ViacheslavBlazhkov/Lesson13/Lesson13.Homework/User.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson13.Homework
+{
+    // class User is not divided into seller and buyer because each user can sell and buy
+    internal class User : Person, IHasInfo
+    {
+        public int UserID { get; private set; }
+        public User(string fname, string lname, string numb) : base(fname, lname, numb)
+        {
+            UserID = new Random().Next(0, 999999);
+        }
+        public static User RegisterUser()
+        {
+            Console.WriteLine("----- REGISTRATION NEW USER -----");
+            Console.Write("Enter first name: ");
+            string fname = Console.ReadLine();
+            Console.Write("Enter last name: ");
+            string lname = Console.ReadLine();
+            Console.Write("Enter phone number: ");
+            string number = Console.ReadLine();
+            Console.WriteLine();
+
+            return new User(fname, lname, number);
+        }
+
+        public Product AddNewProduct()
```

Comment:
Looks like the same as for RegisterUser this method should be static and moved to the Product class